tests/testthat/test-asis-nodes.R

```diff
@@ -1,3 +1,12 @@
+test_that("(#121) single dollar lines dont throw errors", {
+  expected <- "I've only got $2 in the bank. Feels bad, man.\n"
```

Comment:
could you add the same example to the README? saying that you can use dollars as currency without any problem?

Reply:
I've added it, but I've added a caveat that there will be a problem if there is a mix.